frontend/components/events/create/forms/PreviewForm.tsx

```diff
@@ -96,6 +96,7 @@ export const PreviewForm = ({ form, user, imagePreviewUrl, updateField }: Previe
             location={form.location}
             price={form.price}
             vacancy={form.capacity}
+            isPreview={true} // Pass true to make it non-clickable
```

Comment:
I think i twould be better to call this isClickable, so that in the future, if we need to make event card not clickable in other situations we don't have to add a new variable.